java-api-core/src/com/kinvey/java/Constants.java

```diff
@@ -17,6 +17,9 @@
     public static final String AUTH_TOKEN = "authtoken";
     public static final String _KMD = "_acl";
     public static final String _ACL = "_acl";
+
+    public static final String META_ID = "meta.id";
```

Comment:
I'm trying to find where this value is used and why we need it

Reply:
We use this key in classes: `AsyncBatchPushRequest`, `PushBatchRequest`.